modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -500,6 +502,10 @@ private void updateAssignmentInternal(ConfigurationNotificationEvent<byte[]> ass
         CompletableFuture.allOf(futures).join();
     }
 
+    private RaftGroupOptions groupOptionsForInternalTable(InternalTable internalTbl) {
+        return new RaftGroupOptions(internalTbl.storage().isVolatile());
```

Comment:
Is it possible that there is going to be more options inside the RaftGroupOptions? Maybe it's not a good idea to use constructor here and you already have factory methods. Maybe you can add some new method like "RaftGroupOptions#optionsForTable(isVolatile)"? Does it make sense?